### ISSUE TITLE
Captive Portal feature request

### ISSUE THREAD

**Issue by stopspazzing** [ericpaulbishop/gargoyle#576]
Add ability to create a "coffee shop" hotspot.
http://www.gargoyle-router.com/phpbb/viewtopic.php?f=13&t=9190&p=41807&sid=5b8f3bd9c5415e143c7e6fa61bc1a71b#p40653

**Comment by lantis1008** [comment]
It's not a simple request that's why I've largely let those forum posts slide. 
I will look again.

Any implementation would probably base off nodogsplash, at which point why can't people just instal it on their own and have a go.

**Comment by stopspazzing** [comment]
You are right, not too difficult to install: https://wiki.openwrt.org/doc/howto/wireless.hotspot.nodogsplash
Wouldn't a simple plugin make things easier however?

**Comment by lantis1008** [comment]
Integrating with Gargoyles firewall and monitoring is the hardest bit. 
If we accept that as unnecessary then it is largely trivial.

**Comment by stopspazzing** [comment]
How do other open source firmwares do it? Is it because gargoyle has a specific firewall that isn't easy to tweak?

**Comment by lantis1008** [comment]
Because they just tell users to install and self configure lol.

**Comment by stopspazzing** [comment]
Lol. I'm pretty sure that ddwrt and tomato USB by shibby have captive portal of some kind built in. No plugin required

**Comment by lantis1008** [comment]
I would honestly prefer we curbed this idea until the new GUI is finished and integrated. 
Then I will look further.

**Comment by stopspazzing** [comment]
Yes, can this be change to feature request? Thats all I pretty much wanted out of it at this moment
